app-misc/gramps/gramps-4.2.8.ebuild

```diff
@@ -26,7 +26,7 @@ RDEPEND="
 	>x11-libs/gtk+-3.14.8:3[introspection]
 	x11-libs/pango[introspection]
 	x11-misc/xdg-utils
-	reports? ( media-gfx/graphviz )
+	reports? ( media-gfx/graphviz[postscript=] )
```

Comment:
I think you want `postscript?` here. Otherwise, with `USE=-postscript` you're going to force people to build graphviz without postscript.

Reply:
Oh no, I don't want to do that. I want to give people the option because reports still work without postscript, they just can't save to pdf. Is there a way to represent that? Is there a way to have postscript on for graphviz if it's on for gramps, but if it's not on for gramps, still allow it to be on or off for graphviz?

Reply:
Nm, I just realized there's a ? after postscript in your message. You already answered my question.